test/functional/vimscript/system_spec.lua

```diff
@@ -660,4 +660,16 @@ describe('shell :!', function()
         1]])
     end
   end)
+
+  it(':{range}! with powershell without redirecting to buffer', function()
```

Comment:
This test should be done for all shells. I don't know how to do that.
I'm thinking one block _with_ `set_shell_powershell` and one block _without_ it should be enough, but I'm not sure.

Reply:
> one block _with_ `set_shell_powershell` and one block _without_ it should be enough, but I'm not sure.

Yeah that's good enough.